Spanner/src/Database.php

```diff
@@ -536,6 +536,8 @@ public function iam()
      *           up front. Instead, the transaction will be considered
      *           "single-use", and may be used for only a single operation.
      *           **Defaults to** `false`.
+     *     @type array $sessionOptions An array of configuration options used if
```

Comment:
Should we link out to the available options? It looks like you would be able to set labels through here, for instance.